conf/spgwu.bess

```diff
@@ -5,6 +5,28 @@
 import errno
 from conf.utils import *
 
+
+# ====================================================
+#       Parameters
+# ====================================================
+
+
+conf_file = get_env('CONF_FILE', 'conf/spgwu.json')
+conf = get_json_conf(conf_file, False)
+
+# Maximum number of sessions to manage
+max_sessions = conf["max_sessions"]
+
+# Maximum number of flows to manage ip frags for re-assembly
+try:
+    max_ip_frag_flows = conf["max_ip_frag_flows"]
+except ValueError:
+    max_ip_frag_flows = None
+    del max_ip_frag_flows
+    print('Invalid value for max_ip_frag_flows. Not installing IPDefrag module.')
+except KeyError:
+    print('max_ip_frag_flows value not set. Not installing IPDefrag module.')
+
```

Comment:
Can we simplify this and checking below to be if not `None`?

Reply:
This is giving a run-time error. Let's discuss this in detail soon.

Reply:
Fixed!